index.html

```diff
@@ -50,6 +50,7 @@ <h2>Software Developer's Manual</h2>
 <h1>Power</h1>
 
 <p><a href="/d/a48">IBM Power ISA Version 3.0B</a></p>
+<p><a href="/d/4336a7">IBM Power ISA Version 3.1</a></p>
```

Comment:
So far I've only been linking the newest doc from the index.

```suggestion
<p><a href="/d/4336a7">IBM Power ISA Version 3.1</a></p>
```

Reply:
OK. My only concern with that is that 3.0B == Power9 which is what most folks have access to.

Whereas 3.1 == Power10 which is not generally available yet.

v3.1 does have change bars to indicate additions since v3.0B, but they can be easy to miss if you aren't looking for them.

And I realise you can't keep a link to every version of every ISA ever on the homepage.